components/molecule/dropdownOption/src/index.js

```diff
@@ -35,9 +45,14 @@ const MoleculeDropdownOption = ({
 
   const innerClassName = cx([
     CLASS_TEXT,
```

Comment:
I'm not sure this is giving the prior usage of `text-warp` over `withTwoLines` props